implementations/rust/ockam/ockam_command/src/project/addon.rs

```diff
@@ -107,6 +107,10 @@ pub enum ConfigureAddonCommand {
             value_parser(NonEmptyStringValueParser::new())
         )]
         client_id: String,
+
+        /// Attributes to copy from Okta userprofile into Ockam credential.
+        #[arg(long, id = "attributes", value_name = "ATTRIBUTES")]
+        attributes: Vec<String>,
```

Comment:
Will the user always want to pass this argument? If not, it should be an `Option` that we can convert to an empty `Vec` if `None`

Reply:
If it doesn't get any  `--attributes` argument,  it already get an empty Vec.   If the user  pass  `--attributes a  --attributes b`   ,  command gets  a  `["a", "b"]` vector.

Reply:
ooh ok got it! I thought clap would mark it as required